.github/workflows/android_deploy_production.yml

```
@@ -34,6 +26,14 @@ jobs:
           channel: 'stable'
           flutter-version: '3.3.10'
 
+      - name: Generate new project
+        run: |
+          dart pub global activate mason_cli
+          mason get
+          mason make template -c mason-config.json
+          rsync -av --remove-source-files flutter_templates/ ./
+          rm -rf bricks/template
```

I think we should add a comment to explain that we will copy all files and folders in the generated project to the root directory for better clarification
```suggestion
          mason make template -c mason-config.json
          # Move the generated project to the root directory
          rsync -av --remove-source-files flutter_templates/ ./
          rm -rf bricks/template
```

---

@luongvo and to other workflows 😄